kubernetes-sigs/release-notes
# Add 'additional documentation' section

Update all data models to consume the new optional `documentation` field
within the data JSON. The notes component has been adapted as well to
display these information.

---

How to test this:

```
> cp src/testdata/release-notes.json src/assets/data/release-notes-1.16.json
> npm install
> npm start
```

Now the 1.16 alpha release note should show up the `related documentation` section, like this:
![screenshot](https://user-images.githubusercontent.com/695473/61711361-261fff00-ad54-11e9-9fea-713a05e43824.png)

I'm hoping for additional feedback about how to improve the layout. :pray: It is currently not searchable as well, which seems to be something we should do definitely. End-to-end tests are missing, too.

Relates to https://github.com/kubernetes/sig-release/issues/667
